apb.yml

```diff
@@ -45,6 +45,16 @@ plans:
         type: string
         required: False
         title: Postgres Password
+      - name: QUERY_DEPTH_LIMIT
+        description: Limit the complexity of the queries solely by their depth.
+        type: number
+        required: False
+        title: Query depth limit
```

Comment:
Should we maybe provide default value here or mention in the description that if no value is specified, `this value` will be used?

Reply:
true, I will add that now.